portfolio/src/main/java/com/google/sps/data/Comments.java

```diff
@@ -47,7 +43,7 @@ public void addComment(long time, String comment) {
     /**
      * Serialize the comments into JSON format.
      */
-    public String json() {
+    public String stringToJson() {
```

Comment:
Maybe I missed some context here but it will be easer if we could use Gson directly.

Reply:
I am now using Gson to serialize. This `Comments` class  was used to store comments locally. Datastore has replaced this class now.